### PR TITLE
[Tflite] Implement serializing opcode

### PR DESCRIPTION
**Pull request by zhoonit**
- [Tflite] Implement serializing opcode 

```
This patch implements implementing serialize opcode by
`buildOperatorCodes`

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

```

### CODE REVIEW

File: nntrainer/compiler/tflite_interpreter.cpp [hunk 1]
```diff
@@ -331,10 +342,27 @@ buildBuffers(const TfOpIdxMap &map, flatbuffers::FlatBufferBuilder &fbb) {
...
+  const auto &op_codes = map.getIndexMap<tflite::BuiltinOperator>().getData();
+
+  std::vector<flatbuffers::Offset<tflite::OperatorCode>> fb_op_codes;
+  fb_op_codes.reserve(op_codes.size());
+
+  auto create_op_offset = [&fbb](const tflite::BuiltinOperator &op,
+                                 int32_t version = 1) {
+    tflite::OperatorCodeBuilder builder(fbb);
+    builder.add_deprecated_builtin_code(static_cast<int8_t>(op));
+    /// @todo find reason why version field is not shown
```
Comment (kparichay) [inline nntrainer/compiler/tflite_interpreter.cpp:354]: what are the possible values of the version? is it possible to get this version from tflite::interpreter?

**Review by kparichay — APPROVED** [review]
LGTM

File: nntrainer/compiler/tflite_interpreter.cpp (hunk 1 above)
Comment (zhoonit) [reply nntrainer/compiler/tflite_interpreter.cpp:354]: It's been a while after I checked the source code but I remember it was hardcoded to 1 in tf sourcefile as well.

Comment (kparichay) [reply nntrainer/compiler/tflite_interpreter.cpp:354]: then lets keep track to their code if they update it, and let it be 1 for now :)

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Pull request merged**